subsys/net/l2/ppp/Kconfig

```diff
@@ -92,4 +92,11 @@ config NET_L2_PPP_MGMT
 	  Enable support net_mgmt ppp interface which can be used to
 	  configure at run-time ppp drivers and L2 settings.
 
+config NET_L2_PPP_TX_STACK_SIZE
+	int "Stack size for TX handler"
+	default 2048 if COVERAGE_GCOV
+	default 1024
+	help
+	  Set the TX handler stack size.
+
```

Comment:
What is the before/after resource usage with PPP enabled?

Reply:
> What is the before/after resource usage with PPP enabled?

With qemu_x86

Before:
RAM 92056

After:
RAM 92872